middleware/src/middleware_test.go

```diff
@@ -1,15 +1,14 @@
-package middleware_test
+package middleware
 
 import (
 	"testing"
 
-	middleware "github.com/digitalbitbox/bitbox-base/middleware/src"
 	"github.com/digitalbitbox/bitbox-base/middleware/src/rpcmessages"
 	"github.com/stretchr/testify/require"
 )
 
 // setupTestMiddleware middleware returns a middleware setup with testing arguments
-func setupTestMiddleware() *middleware.Middleware {
+func setupTestMiddleware() *Middleware {
 	argumentMap := make(map[string]string)
 	argumentMap["bitcoinRPCUser"] = "user"
```

Comment:
unrelated, but for another PR/issue: this should probably be a struct with well defined fields, otherwise changing the config map probably will not be applied consistently in the future.

Reply:
Agree. https://github.com/shiftdevices/bitbox-base-internal/issues/299